Solutions/Lookout/Data Connectors/azuredeploy_Connector_LookoutAPI_AzureFunction.json

```diff
@@ -0,0 +1,269 @@
+                "[resourceId('Microsoft.Insights/components', variables('FunctionName'))]"
+            ],
+            "kind": "functionapp,linux",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "name": "[variables('FunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true,
+                "siteConfig": {
+                    "linuxFxVersion": "python|3.8"
+                }
+            },
+            "resources": [
+                {
+                    "apiVersion": "2018-11-01",
+                    "type": "config",
+                    "name": "appsettings",
+                    "dependsOn": [
+                        "[concat('Microsoft.Web/sites/', variables('FunctionName'))]",
+                        "[resourceId('Microsoft.KeyVault/vaults/', variables('KeyVaultName'))]",
+                        "[resourceId('Microsoft.KeyVault/vaults/secrets', variables('KeyVaultName'), variables('EnterpriseName'))]",            
+                        "[resourceId('Microsoft.KeyVault/vaults/secrets', variables('KeyVaultName'), variables('ApiKey'))]"
+                    ],
+                    "properties": {
+                        "FUNCTIONS_EXTENSION_VERSION": "~3",
+                        "FUNCTIONS_WORKER_RUNTIME": "python",
+                        "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]",
+                        "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]",
+                        "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                        "WorkspaceID": "[parameters('WorkspaceID')]",
+                        "WorkspaceKey": "[parameters('WorkspaceKey')]",
+                        "EnterpriseName": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('EnterpriseName')).secretUriWithVersion, ')')]",
+			            "ApiKey": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ApiKey')).secretUriWithVersion, ')')]",
+                        "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
+                        "KeyVaultName" : "[variables('KeyVaultName')]",
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/gate6/Azure-Sentinel/blob/master/Solutions/Lookout/Data%20Connectors/LookoutAPISentinelConnector.zip?raw=true"
```

Comment:
Please replace value of WEBSITE_RUN_FROM_PACKAGE with
[https://aka.ms/sentinel-lookoutapi-functionapp](https://aka.ms/sentinel-lookoutapi-functionapp)

Reply:
Done